src/components/shared/ContributionsTracker.tsx

```diff
@@ -56,9 +63,16 @@ const ContributionsTracker = () => {
         showMonthLabels={true}
       />
       <div className="flex justify-between">
-        <p className="text-subtitle text-myWhite-300">
-          Learn how we count contributions
-        </p>
+        <TooltipProvider>
+          <Tooltip>
+            <TooltipTrigger className="flex justify-between">
+              <p className="text-subtitle text-myWhite-300">
+                Learn how we count contributions
+              </p>
+            </TooltipTrigger>
+            <TooltipContent>Hover</TooltipContent>
+          </Tooltip>
+        </TooltipProvider>
```

Comment:
This was mostly as a demo for the tooltip itself- so this is my bad. But at the moment it doesn't make much sense for this section to just say "hover" as a tooltip.